tests/DCPS/DPFactoryQos/run_test.pl

```diff
@@ -12,11 +12,19 @@
 use strict;
 
 my $test = new PerlDDS::TestFramework();
+my $is_rtps_disc = 0;
```

Comment:
These changes in run_test.pl shouldn't be needed.  The Perl module for TestFramework knows about rtps_disc and make sure setup_discovery() does the right thing.